src/test/kotlin/io/bazel/kotlin/KotlinAssertionTestCase.kt

```diff
@@ -120,7 +120,7 @@ abstract class BasicAssertionTestCase {
         ProcessBuilder().command("bash", "-c", Paths.get(executable).fileName.toString())
             .also { it.directory(executable.resolveDirectory()) }
             .start().let {
-                it.waitFor(5, TimeUnit.SECONDS)
+                it.waitFor(10, TimeUnit.SECONDS)
```

Comment:
Why this change?

Reply:
I hope "friends" are not making it half as fast ;)

Reply:
No, I added some time-delay in the dagger example, using coroutines, and that causes the dagger example to run a few seconds long by design. This just pads it.